app/containers/AccountProvider/saga.js

```diff
@@ -312,10 +318,17 @@ export function* isAvailableAction(isValid, data = {}) {
       return true;
     }
 
+    const path = window.location.pathname + window.location.hash;
     if (
       profileInfo.permissions?.find(
         x => x.value == COMMUNITY_ADMIN_VALUE && x.community == communityID,
-      )
+      ) ||
+      (path === routes.communitiesCreateByInvite(profileInfo.user) &&
```

Comment:
I think it can be better to add function `routes.isCommunitiesCreateByInvite(profileInfo.user)` to routes that checks if current path matches community create invite. This way logic related to routes is kept in a single place